pom.xml

```diff
@@ -71,6 +71,8 @@
         <dependency.gson.version>2.8.9</dependency.gson.version>
         <dependency.httpclient.version>4.5.13</dependency.httpclient.version>
         <dependency.httpcore.version>4.4.16</dependency.httpcore.version>
+        <dependency.httpcomponents-httpclient5.version>5.2.1</dependency.httpcomponents-httpclient5.version>
```

Comment:
Can we use one property for both dependencies?

Reply:
They tended to differ in the past (httpcore vs httpclient) so I would suggest leaving as it is.